src/Robo/Commands/Deploy/DeployCommand.php

```diff
@@ -162,6 +162,13 @@ public function checkDirty(array $options = ['ignore-dirty' => FALSE]) {
         $this->logger->warning("There are uncommitted changes on the source repository.");
       }
       else {
+        if ($options['verbose']) {
+          $this->taskExec('git diff --exit-code')
```

Comment:
I'd say `git status` is less verbose. In this case you probably care most about what files are changed, not the diff.

Reply:
Hi @alexxed !
Thank you  for your review.
Yes `git status` is less verbose. 
Therefore, it is not always clear what exactly has changed. I gave an example with settings.php
There are several behaviors that can lead to changes in the build.
- If a Drupal installer is running, a line with the type of the installed profile may be added.
- File permissions may change.
- A line with blt is added to the end of the file

Not all such changes are understandable when viewing git status.
In addition, when an error is found, more information gives more power to solve it.

Reply:
Looking at the screenshot I see it's a CI, so it makes sense to get as much info as possible automatically.

Reply:
@alexxed great, thanks!
What is the next step to processed it?